tests/lexer/lexer_test.cpp

```diff
@@ -1,13 +1,122 @@
 #include "catch2/catch.hpp"
 #include "lexer.hpp"
+#include "token.hpp"
+#include <string>
+#include "fmt/format.h"
+#include "trie.hpp"
+#include "catch_helpers.hpp"
```

Comment:
Put catch_helpers right after catch.hpp to emphasize that the order _is_ important in this case.